ofxProjectGenerator/src/addons/ofAddon.cpp

```diff
@@ -310,7 +315,7 @@ void ofAddon::parseVariableValue(string variable, string value, bool addToValue,
 	}
 
 	if(variable == "ADDON_DATA"){
-		addReplaceStringVector(data,value,"",addToValue);
+		addReplaceStringVector(data,value,addonRelPath,addToValue);
```

Comment:
what is this? doesn't seem related to any of the problems explained in the PR no?

Reply:
No, sorry I don't really know why that happened.

Reply:
ok, fixed it.